locales/sv-SE.json

```diff
@@ -6,172 +6,92 @@
   "instructions": {
     "arrive": {
       "phrases": {
-        "0": "Ankomst: <TIME>.",
-        "1": "Ankomst: <TIME> till <TRANSIT_STOP>."
-      },
-      "example_phrases": {
-        "0": [
-          "Ankomst: 8:02 AM."
-        ],
-        "1": [
-          "Ankomst: 8:02 AM till 8 St - NYU."
-        ]
+        "0": "Anländer: <TIME>.",
+        "1": "Anländer: <TIME> till <TRANSIT_STOP>."
       }
     },
     "arrive_verbal": {
       "phrases": {
-        "0": "Ankomst vid <TIME>.",
-        "1": "Ankomst vid <TIME> till <TRANSIT_STOP>."
-      },
-      "example_phrases": {
-        "0": [
-          "Ankomst vid 8:02 AM."
-        ],
-        "1": [
-          "Ankomst vid 8:02 AM till 8 St - NYU."
-        ]
+        "0": "Anländer <TIME>.",
+        "1": "Anländer <TIME> till <TRANSIT_STOP>."
       }
     },
     "bear": {
       "phrases": {
-        "0": "Håll <RELATIVE_DIRECTION>.",
-        "1": "Håll <RELATIVE_DIRECTION> till <STREET_NAMES>.",
-        "2": "Håll <RELATIVE_DIRECTION> till <BEGIN_STREET_NAMES>. Fortsätt på <STREET_NAMES>.",
-        "3": "Håll <RELATIVE_DIRECTION> för att stanna kvar på <STREET_NAMES>.",
-        "4": "Bear <RELATIVE_DIRECTION> at <JUNCTION_NAME>.",
-        "5": "Bear <RELATIVE_DIRECTION> toward <TOWARD_SIGN>."
+        "0": "Sväng <RELATIVE_DIRECTION>.",
+        "1": "Sväng <RELATIVE_DIRECTION> in på <STREET_NAMES>.",
+        "2": "Sväng <RELATIVE_DIRECTION> in på <BEGIN_STREET_NAMES>. Fortsätt på <STREET_NAMES>.",
+        "3": "Sväng <RELATIVE_DIRECTION> för att stanna kvar på <STREET_NAMES>.",
+        "4": "Sväng <RELATIVE_DIRECTION> i <JUNCTION_NAME>.",
+        "5": "Sväng <RELATIVE_DIRECTION> mot <TOWARD_SIGN>."
       },
-      "empty_street_name_labels": [
```

Comment:
i see the sorting is different than before. we have a script in the locales directory called `canonisize.py`. you can run it on this file to resort the json. that will not only make it so that the json keeps the same structure as all the other json translations but also it will make this diff a lot easier to read!